tee-worker/service/src/main.rs

```diff
@@ -896,6 +939,7 @@ fn check_we_are_primary_validateer(
 	Ok(enclave_count_of_previous_block == 0)
 }
 
+#[allow(clippy::unwrap_used)]
```

Comment:
Can we have better panic messages for these unwraps, This is usually caused when the `config.json` is not according to the format we expect, So a panic message mentioning exactly that could be more helpful.